packages/stylelint-config/package.json

```
@@ -13,8 +13,8 @@
     "linter"
   ],
   "peerDependencies": {
-    "stylelint": "^13.8",
-    "stylelint-config-standard": "^20"
+    "stylelint": "^15.10.3",
```

It's better to specify it as `^15.10` to support a wider range of versions. Currently, if a user will have `stylelint@15.10.0` in their dependencies, they would end up with two versions of `stylelint`.


```suggestion
    "stylelint": "^15.10",
```